includes/class-wcpdf-settings-general.php

```diff
@@ -247,6 +250,29 @@ public function init_settings() {
 		return;
 	}
 
+	public function text_element_callback( $args ) {
```

Comment:
what does this do that the text_input doesn't do? Would it be something to add to the default text_input?

Reply:
How should work? I tested and didn't work